utils/mil_ros_tools/online_bagger/config/topics_to_stream.yaml

```diff
@@ -0,0 +1,13 @@
+# list an arbitrary set of topics to  subscribe to and stream:
+  # ram_limit is not yet implemented
+
+stream_time: 30 # seconds
+      # down_stream_rate: 10 Hz 
+ram_limit: 0.5 # gb 
+message_topics: ["/odom", 
```

Comment:
just "topics " would probably be a better name for this parameter.

Also, because this is going in the common repo, perhaps this config and the launch file should be renamed to "online_bag_example.yaml and online_bag_example.launch" as individual projects will probably create their own configs.